langlib/langlib-test/src/test/resources/test-src/regexp_test.bal

```diff
@@ -994,6 +994,78 @@ function testSplit() {
     string[] arrExpected9 = ["1", "2", "3", "4", "5"];
     assertEquality(5, resArr9.length());
     assertEquality(arrExpected9, resArr9);
+
+    string str10 = "1 2   3 4  5";
+    string[] resArr10 = re `${""}`.split(str10);
+    string[] arrExpected10 = ["1"," ","2"," "," "," ","3"," ","4"," "," ","5"];
+    assertEquality(12, resArr10.length());
+    assertEquality(arrExpected10, resArr10);
+
+    string str11 = "1 2   3 4  5";
+    string[] resArr11 = re `()`.split(str11);
+    string[] arrExpected11 = ["1"," ","2"," "," "," ","3"," ","4"," "," ","5", ""];
```

Comment:
According to Javascript this is an incorrect result.
The corect result should be 

```
[
  '1', '',  ' ', '',  '2', '',  ' ',
  '',  ' ', '',  ' ', '',  '3', '',
  ' ', '',  '4', '',  ' ', '',  ' ',
  '',  '5'
]
```

Reply:
Created an issue for this and discussed to remain it as same for now. https://github.com/ballerina-platform/ballerina-lang/issues/39826